src/constants.ts

```diff
@@ -8,7 +8,9 @@ export let SUPERRARE_V2 = Address.fromString("0xb932a70A57673d89f4acfFBE830E8ed7
 
 export let KNOW_ORIGIN = Address.fromString("0xFBeef911Dc5821886e1dda71586d90eD28174B7");
```

Comment:
@developerfred @frostiq 
There is an error. Missing `d`. Correct contract address is 0xFBeef911Dc5821886e1dda71586d90eD28174B7d

Reply:
Thanks @Tibike6!

Reply:
Ow, that's probably why I got an error during graph resync
Thanks for noticing @Tibike6!